src/Symfony/Component/Routing/Tests/Loader/DependencyInjection/ServiceRouterLoaderContainerTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Tests\Loader;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\Routing\Loader\DependencyInjection\ServiceRouterLoaderContainer;
+
+class ServiceRouterLoaderContainerTest extends TestCase
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * @var ContainerInterface
+     */
+    private $serviceLocator;
+
+    /**
+     * @var ServiceRouterLoaderContainer
+     */
+    private $serviceRouterLoaderContainer;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp(): void
```

Comment:
breaks with phpunit before 8 + php before 7.1: void should be removed I think

Reply:
all of them :)

Reply:
I will remove it because there are no other instances of this in the code, but for my knowledge, how could that break with PHPUnit < 8 ? Because I actually always add strict return types even when parent / interface does not have it yet and I never had problems 😕 

For the PHP version I think it's fine since the minimum supported version on master is 7.1.3.

Reply:
> add strict return types even when parent / interface does not have it yet and I never had problems

because you run this with PHP 7.2 - but using PHP 7.1, that would break, isn't it?

Reply:
It doesn't break (https://3v4l.org/QbQ6a) but it's not important.